app/src/main/java/io/horizontalsystems/bankwallet/core/managers/FeeRateProvider.kt

```diff
@@ -5,38 +5,37 @@ import io.horizontalsystems.bankwallet.core.FeeRatePriority
 import io.horizontalsystems.bankwallet.core.IAppConfigProvider
 import io.horizontalsystems.bankwallet.core.IFeeRateProvider
 import io.horizontalsystems.bankwallet.entities.FeeRateInfo
-import io.horizontalsystems.feeratekit.FeeRate
 import io.horizontalsystems.feeratekit.FeeRateKit
+import io.horizontalsystems.feeratekit.model.FeeProviderConfig
+import io.horizontalsystems.feeratekit.model.FeeRate
+import io.reactivex.Single
 
 class FeeRateProvider(context: Context, appConfig: IAppConfigProvider) : FeeRateKit.Listener {
 
-    private val feeRateKit by lazy {
-        FeeRateKit(
-                infuraProjectId = appConfig.infuraProjectId,
-                infuraProjectSecret = appConfig.infuraProjectSecret,
-                context = context,
-                listener = this
-        )
+    private val feeRateKit = FeeRateKit(
+            FeeProviderConfig(infuraProjectId = appConfig.infuraProjectId,
+                              infuraProjectSecret = appConfig.infuraProjectSecret,
+                              btcCoreRpcUrl = appConfig.btcCoreRpcUrl,
+                              btcCoreRpcUSer =  appConfig.btcCoreRpcUser,
+                              btcCoreRpcPassword = appConfig.btcCoreRpcPassword),
+            context = context,
```

Comment:
indentation from context is different, from top items

Reply:
Because base parameters are :
1.FeeProviderConfig
2.Context
3. Listener
They are all in one indent line.